Include float.h for DBL_EPSILON

https://bugs.gentoo.org/show_bug.cgi?id=480272

Patch was written by Jeroen Roovers <jer@gentoo.org>
--- a/src/handlers.c
+++ b/src/handlers.c
@@ -13,6 +13,7 @@
 #include "all.h"
 
 #include <time.h>
+#include <float.h>
 #include <sys/time.h>
 #include <xcb/randr.h>
 #include <X11/XKBlib.h>
